js/views/app_view.js

```diff
@@ -54,7 +54,7 @@
         },
         finishLightImport: function() {
           var options = {
-            startStep: Whisper.InstallView.Steps.SCAN_QR_CODE,
+            postImport: true
```

Comment:
Is this _post_ as in _after_? This wasn’t clear to me at first glance. How about `hasExistingData: true` or similar?